### PR TITLE
Issu11

### PR DESCRIPTION
**Pull request by warren6688**
## Change Summary 📝
[Briefly summarise the changes that you made.]

---update issu11 single line, ready to merge 

## Change Form 📋
Please mark as NA if not applicable. If a certain criterion is not met, kindly provide a reason.

- [ ] The pull request title has an issue number.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] Relevant tests (if applicable) have been added or updated.

---

## Any background context you want to provide? 🖼️

Any additional context or screenshots that could help the reviewers?

## Reviewers 🕵️

(Optional) Suggest reviewers that might be interested in this PR.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,11 +1,22 @@
...
+venv/lib/python3.11/site-packages/torch/_namedtensor_internals.py
+venv/lib/python3.11/site-packages/torch/_ops.py
+venv/lib/python3.11/site-packages/torch/_python_dispatcher.py
+venv/lib/python3.11/site-packages/torch/_sources.py
+venv/lib/python3.11/site-packages/torch/_storage_docs.py
+venv/lib/python3.11/site-packages/torch/_tensor_docs.py
 # Python cache files
 __pycache__/
-app/pycache/
-__pycache__
```
Comment (yylocky) [inline .gitignore:20]: These two shouldn't be deleted, probably your branch is outdated, try pull from main branch:)

File: app/routes.py [hunk 2]
```diff
@@ -1,12 +1,15 @@
 from flask import render_template, request, redirect, url_for, flash, jsonify, current_app, session
 from flask_login import current_user, login_user, logout_user, login_required
 from app import app, db, forms
-from app.models import Login, User, WorkloadAllocation, Role, Work
+from app.models import Login, User, Role
+
+from app.models import WorkloadAllocation
+from app.models import Department,Work
 from app.forms import LoginForm, SignupForm
```
Comment (yylocky) [inline app/routes.py:8]: Minor one tho, you can import all the models in online.

File: app/routes.py
Comment (yylocky) [inline app/routes.py:1]: Sorry I couldn't go through each line, cause I think you are misunderstanding the work, and trying to delete all the other members' work. be careful when you are resolve any conflicts. you can add more endpoints for your function, but please try not to delete other members' work.

File: app/templates/.DS_Store
Comment (yylocky) [inline app/templates/.DS_Store:1]: If you can get the update gitignore, this file will be ignored.

File: app.db
Comment (yylocky) [inline app.db:1]: Please update the db first, refer to readme file. Also, Like we discussed, you can add test data but try do not edit the existing data.

File: app/models.py
Comment (yylocky) [inline app/models.py:1]: Again update db model first.

**Review by yylocky — COMMENTED** [review]
Can see a lot of functions been implemented well, but still when try to integrate with main please be careful resolving any conflict and always keep you main branch updated. Thanx!

**Comment by warren6688** [comment]
> Can see a lot of functions been implemented well, but still when try to integrate with main please be careful resolving any conflict and always keep you main branch updated. Thanx!

**Comment by warren6688** [comment]
> > Can see a lot of functions been implemented well, but still when try to integrate with main please be careful resolving any conflict and always keep you main branch updated. Thanx!

Cheers, Mira, thank you for the comments, really useful, real appreciate it.

**Pull request closed without merging**